pygmt/enums.py

```diff
@@ -0,0 +1,39 @@
+"""
+Enumerations for PyGMT.
+"""
+
+from enum import IntEnum
+
+
+class GridFormat(IntEnum):
+    """
+    Enum for the GMT grid format ID.
+
+    These enums are defined in 'gmt_grdio.h'.
+    """
+
+    UNKNOWN = 0  #: Unknown grid format
```

Comment:
`#: ` is the special way to document attributes, following https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-autoproperty.